packages/tabs/src/styled/StyledTab.ts

```diff
@@ -5,55 +5,102 @@
 
 const COMPONENT_ID = 'tabs.tab';
 
 interface IStyledTabProps {
-  isHovered?: boolean;
-  isFocused?: boolean;
-  isActive?: boolean;
   disabled?: boolean;
   isSelected?: boolean;
 }
 
+/**
+ * A high specificity is needed to apply the border-color in vertical orientations
+ */
+const colorStyles = ({ theme, isSelected }: IStyledTabProps & ThemeProps<DefaultTheme>) => css`
+  color: ${isSelected ? getColor('primaryHue', 600, theme) : 'inherit'};
+
+  &&& {
+    border-color: ${isSelected && 'currentColor'};
+  }
+
+  &:hover {
+    color: ${props => getColor('primaryHue', 600, props.theme)};
+  }
+
+  &:focus {
+    outline: none;
+    text-decoration: none;
+  }
+
+  &:active {
+    border-color: currentColor;
+    color: ${props => getColor('primaryHue', 600, props.theme)};
+  }
+
+  &::before {
+    transition: box-shadow 0.1s ease-in-out;
+    content: '';
+  }
+
+  &:active::before {
+    box-shadow: none;
+  }
+`;
+
+const focusStyles = ({ theme }: ThemeProps<DefaultTheme>) => css`
+  &[data-garden-focus-visible] {
+    color: ${getColor('primaryHue', 600, theme)};
+  }
+
+  &[data-garden-focus-visible]::before {
+    position: absolute;
+    top: ${theme.space.base * 2.5}px;
+    right: ${theme.space.base * 6}px;
+    left: ${theme.space.base * 6}px;
+    border-radius: ${theme.borderRadii.md};
+    box-shadow: inset ${theme.shadows.sm(getColor('primaryHue', 600, theme, 0.35)!)};
+    height: ${theme.space.base * 5}px;
+    pointer-events: none;
+  }
+`;
+
 /**
  * Accepts all `<div>` props
  */
 export const StyledTab = styled.div.attrs<IStyledTabProps>(props => ({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
-  className: classNames(TabStyles['c-tab__list__item'], {
-    // Visual states
-    [TabStyles['is-hovered']]: props.isHovered,
-    [TabStyles['is-focused']]: props.isFocused,
-    [TabStyles['is-active']]: props.isActive,
-    [TabStyles['is-disabled']]: props.disabled,
-    [TabStyles['is-selected']]: props.isSelected
-  })
+  'aria-disabled': props.disabled
 }))<IStyledTabProps>`
-  /** This removes focus styling when not needed. Mimics :focus-visible */
-  &&& {
-    ${props =>
-      !props.isFocused &&
-      `
-      :before {
-        box-shadow: none;
-      }
-    `}
+  display: inline-block;
+  position: relative;
+  transition: color 0.25s ease-in-out;
+  border-bottom: ${props => props.theme.borderStyles.solid} transparent;
+  border-width: ${props => props.theme.borderWidths.md};
+  cursor: pointer;
+  padding: ${props => props.theme.space.base * 2.5}px ${props => props.theme.space.base * 7}px
+    ${props => props.theme.space.base * 1.5}px;
+  overflow: hidden;
+  vertical-align: top;
+  user-select: none;
+  text-align: center;
+  text-decoration: none;
+  text-overflow: ellipsis;
+
+  ${colorStyles}
+  ${focusStyles}
+
+  &[aria-disabled='true'] {
+    border-color: transparent;
+    cursor: default;
+    color: ${props => getColor('neutralHue', 400, props.theme)};
```

Comment:
Move this color-related value up to the `colorStyles` function (along with `border-color` above). Leave `cursor` here.

Reply:
I've split this style definition.